.github/workflows/golang.yaml

```diff
@@ -20,8 +20,3 @@ jobs:
         run: make integration
       - name: Test Coverage
         run: make coverage
-      - name: Upload Coverage
```

Comment:
q: this will result in missing quality gate. Codecov reports on coverage on the PR. We need an alternative before we remove this. Can we discuss before merging?

Reply:
merging for now, per our discussion earlier. We don't have anywhere to upload in codecov anymore